docs/quickstarts/openshift-ai-rhods-launch/openshift-ai-rhods-launch.yml

```diff
@@ -0,0 +1,22 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: openshift-ai-rhods-launch
+  externalDocumentation: true
+  annotations:
+    order: 1
+  labels:
+    service-category/data-services: "true"
+    product/rhods: "true"
+spec:
+  version: 0.1
+  type:
+    text: Documentation
+    color: orange
+    displayName: Learn how to launch Red Hat OpenShift Data Science
```

Comment:
The `displayName` should be an attribute of the `spec` object. (There are two extra spaces that are not required)

Reply:
Many thanks for the info @Hyperkid123  -- fixed now. 
Should I have been able to see that CI error message myself?